blaze-client/src/main/scala/org/http4s/client/blaze/BlazeClientBuilder.scala

```diff
@@ -20,6 +23,7 @@ sealed abstract class BlazeClientBuilder[F[_]] private (
     val responseHeaderTimeout: Duration,
     val idleTimeout: Duration,
     val requestTimeout: Duration,
+    val connectingTimeout: Duration,
```

Comment:
Bikeshed: async-http-client and jetty-client both call this `connectTimeout`.  Maybe that's a better name for consistency.  Would also affect the blaze PR.